File: program/Primes/Primes.js
Object.values(
    [100].flatMap(n=>[...Array(Math.trunc(Math.sqrt(n))).keys()]
        .map(x=>x+1)
        .flatMap(x=>[...Array(Math.trunc(n/x-1)).keys()]
            .map(y=>x*(y+2))))
    .reduce((x,y)=>{x[y]=x[y]??[];x[y].push(y);return x},{}))
.filter(x=>x.length==1)
.flat()
.forEach(x=>console.log(x))

File: program/Primes/Primes.java
import java.util.stream.IntStream;
import java.util.stream.Collectors;
import java.util.stream.Stream;

class Primes {
    public static void main(String[] args) {
        Stream.of(100)
            .flatMap(n->IntStream.range(1,(int)Math.sqrt(n)+1)
                    .boxed()
                    .flatMap(x->IntStream.range(2,n/x+1)
                        .boxed()
                        .map(y->x*y)))
        .collect(Collectors.groupingBy(x->x))
        .entrySet()
        .stream()
        .filter(x->x.getValue().size()==1)
        .map(x->x.getKey())
        .forEach(x->System.out.println(x));
    }
}
